CHANGELOG.txt

```diff
@@ -1,5 +1,59 @@
 == Changelog ==
 
+= 4.0.0 - 2020-03-03
```

Comment:
IMO, we should only add the changelog for 4.0 to the CHANGELOG.txt file after the final release, and I believe that is what we did in previous versions. CHAGENLOG.txt in the master branch is where we point everyone for a full version of the changelog, including information about a version that is yet to be released could cause some confusion.